composer.json

```diff
@@ -21,16 +21,18 @@
     ],
     "require": {
         "php": ">=5.3.2",
-        "symfony/framework-bundle": "~2.3",
-        "symfony/security-bundle": "~2.3"
+        "symfony/framework-bundle": "~2.6|~3.0",
+        "symfony/http-kernel": "~2.6|~3.0",
+        "symfony/security-bundle": "~2.6|~3.0",
+        "symfony/form": "~2.6|~3.0"
```

Comment:
Forbidding Symfony versions before 2.6 is a BC break that cannot be done in a minor version release (not to talk about patch releases).